CMakeLists.txt

```diff
@@ -150,8 +150,18 @@ set(RDKit_ExternalDir "${CMAKE_CURRENT_SOURCE_DIR}/External")
 set(RDKit_DataDir "${CMAKE_CURRENT_SOURCE_DIR}/Data")
 
 #include catch
-add_subdirectory(${RDKit_ExternalDir}/catch)
-message("CATCH: ${CATCH_INCLUDE_DIR}")
+find_package(catch2 3)
```

Comment:
This one needs to be `find_package(Catch2 3)`  (capitalized c) for me in order for CMake to pick things up, see `https://github.com/catchorg/Catch2/blob/devel/CMake/Catch2Config.cmake.in`

Reply:
> This one needs to be `find_package(Catch2 3)` (capitalized c) for me in order for CMake to pick things up, see `https://github.com/catchorg/Catch2/blob/devel/CMake/Catch2Config.cmake.in`

yep, fixed that typo